app/views/groups/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class='new--cart'>
+  <h2 class='cartegory--title'>Categories</h2>
+  <div class='btn--div'>
+    <%= button_to "Logout", destroy_user_session_path, method: :delete, class: "btn btn-secondary btn-block btn-lg" %>
+  </div>
+</div>
+
+<div class='categories--index'>
+  <div id="groups--list">
+    <% @groups.each do |group| %>
+      <div class='group--card'>
+        <div class='group--details'>
+          <div class='group--icon'>
+            <%= image_tag group.icon, height: 50 %>
+          </div>
+          <p class='group--name'><%= group.name %></p>
+          <p class='group--amount'>$ <%= group.total_amount %></p>
+        </div>
+        <p class='group--link'><%= link_to "See all #{group.name} transactions", group_entities_path(group_id: group.id) %></p>
+      </div>
+    <% end %>
```

Comment:
- [ ] Kindly add the date which each category is created as seen in the design guideline. You can simply make use of the `created_at` column that is present in the group's table and format it to date format as seen in the guideline. You can use this [ruby-date-format-strftime/cheatsheet](https://www.shortcutfoo.com/app/dojos/ruby-date-format-strftime/cheatsheet)

<img width="141" alt="chrome_zNI8J7ubZU" src="https://user-images.githubusercontent.com/81468675/171645472-f5eb8821-aa28-462e-bb65-6b9298b9bab2.png">

Reply:
✅ **Resolved**
I have implemented a feature for displaying time of category creation